src/App.tsx

```diff
@@ -123,8 +123,12 @@ function App() {
           selectedTabId={selectedTabId}
           tabs={tabsList}
         ></Tab>
-        {selectedTabId === 'issues' && <IssuesList />}
-        {selectedTabId === 'commits' && <CommitsList />}
+        {selectedRepo !== null && selectedTabId === 'issues' && (
+          <IssuesList repo={selectedRepo} />
+        )}
+        {selectedRepo !== null && selectedTabId === 'commits' && (
```

Comment:
There's no need for `selectedRepo !== null &&` here, because you're not passing `selectedRepo`  to the `<CommitsList>`.